tests/atest/transformers/SplitTooLongLine/expected/feed_until_line_length.robot

```diff
@@ -0,0 +1,152 @@
+# Use 4 spaces separator and line_length=80
+
+*** Tasks ***
+
+Different keyword calls
+    This is a keyword     fits even with its    # comment
+
+    # comment, but has bad spacing
+    This is a keyword    fits with its
+
+    This is a keyword    these fit    but    only if you space them correctly
+
+    This is a keyword    these args do not fit
+    ...    even if you set spacing properly
+
+    This is a keyword    this    last    argument    is    not    really
+    ...    a # comment
+
+    # comment
+    This is a keyword    these    arguments    wont    fit    with    that
+
+    # comment
+    This is a keyword    these    arguments    wont    fit    with    or
+    ...    without    that
+
+    # Edge case here →→→→→→→→→→→→→→→→                                    HERE
+    This is a keyword    these    args    have    an    interesting    ${EMPTY}
+    ...    More arguments here
+
+    This is a keyword     and     these      are       its     args
+    ...    here   are   some    more    args      to      split
+    ...    with                irregular                       spacing
```

Comment:
Do we want to save irregular spacing or rather overwrite it with default one?

Reply:
This implementation operates on a "doesn't need fixing - don't fix" assumption, so if a call doesn't go beyond `line_length` columns - it is not reformatted. I guess reformatting of spacing can be done in a different transformer?

Reply:
Yes you're right, that's actually good (with one transformation - one responsibility approach).